benchmark/src/main/scala/org/virtuslab/akkasaferserializer/benchmark/Benchmark.scala

```diff
@@ -0,0 +1,79 @@
+package org.virtuslab.akkasaferserializer.benchmark
+
+import akka.actor.testkit.typed.scaladsl.{ActorTestKit, SerializationTestKit}
+import akka.actor.typed.ActorSystem
+import com.typesafe.config.ConfigFactory
+import org.virtuslab.akkasaferserializer.benchmark.model.{Adt, Primitive, Sequence}
+
+import java.io.PrintWriter
+
+object Benchmark {
+
+  private final val warmupIterations = 100_000
+
+  private final val testIterations = 10_000_000
+
+  def benchmark(configName: String, output: String): Unit = {
+    new PrintWriter(output) {
+      write(primitive(configName).toString)
+      write("\n")
+      write(adt(configName).toString)
+      write("\n")
+      write(sequence(configName).toString)
+      write("\n")
+      close()
+    }
+  }
+
+  private def primitive(configName: String): Long = {
+    val testKit = init(configName)
+    val obj = Primitive()
+    for (_ <- 1 to warmupIterations) {
```

Comment:
Hmmm why specifically these `warmupIterations`? I'd understand *one* iteration would be useful to sift out overheads due to classloading etc... but why 100k? Pls comment in code